src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadService.kt

```diff
@@ -385,6 +397,8 @@ class ComCaseloadService(
     )
   }
 
+  private fun ManagedCase.findRelevantLicence() = licences.find { licence -> licences.size == 1 || licence.licenceStatus != ACTIVE }
```

Comment:
hate all this file 😅 ... extracted out this helper as the logic was being used in a few places across create and vary... 

This could do with a bit of TLC